java/README.md

```diff
@@ -80,8 +80,12 @@ variable are set, the system property takes precedence.
 
 ## Java Properties
 
-For java 9 or later, should set "-Dio.netty.tryReflectionSetAccessible=true".
+ * For java 9 or later, should set "-Dio.netty.tryReflectionSetAccessible=true".
 This fixes `java.lang.UnsupportedOperationException: sun.misc.Unsafe or java.nio.DirectByteBuffer.(long, int) not available`. thrown by netty.
+ * To support duplicate fields in a `StructVector` enable "-Darrow.struct.conflict.policy=CONFLICT_APPEND".
```

Comment:
we've been using the policy that java properties should also be settable via environment variable (I believe in some context environment variables are easier to set).

Reply:
done!